### PR TITLE
[ADD]middleware for FastAPI

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

File: src/pybrake/middleware/fastapi.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            _before_request(request)
+        response = await call_next(request)
+        if config.get("performance_stats"):
+            _after_request(response, notifier)
+        return response
+
+    # Route breakdown stats monitoring
+    old_render = Response.render
+
+    def patch_render(self, content) -> bytes:
```
Comment (kyrylo) [inline src/pybrake/middleware/fastapi.py:174]: Why do we annotate the return type? We didn't do that in previous PRs.

File: src/pybrake/middleware/fastapi.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                start_time=metric.start_time,
+                end_time=time.time(),
+            )
+
+    return _sqla_after_cursor_execute
+
+
+def init_app(app: FastAPI, sqlEngine: future.Engine = None) -> FastAPI:
+    """
+    Initiate PyBrake notifier and apply patch for Error monitoring and APM.
```
Comment (kyrylo) [inline src/pybrake/middleware/fastapi.py:126]:
```suggestion
    Initiate the pybrake notifier and apply the patch for error monitoring and APM.
```

**Review by kyrylo — APPROVED** [review]

File: src/pybrake/middleware/fastapi.py (hunk 1 above)
Comment (smurf-U) [reply src/pybrake/middleware/fastapi.py:174]: Actually, I added a patch here and created a new method in the same way as method define in framework. This is also good practise. So...

**Comment by smurf-U** [comment]
PTAL

**Comment by kyrylo** [comment]
I already approved this PR.

**Pull request merged**